core/test/financial_templates/PricelessPositionManager.js

```diff
@@ -311,6 +345,93 @@ contract("PricelessPositionManager", function(accounts) {
     assert(await didContractThrow(pricelessPositionManager.transfer(accounts[3], { from: sponsor })));
   });
 
+  it("Redemption post expiry", async function() {
```

Comment:
Just FYI: I think this is a really solid test: very clean, well commented, and understandable 💯